bats/tests/helpers/vm.bash

```diff
@@ -60,9 +60,11 @@ start_container_engine() {
     local args=(
         --application.debug
         --application.updater.enabled=false
-        --container-engine.name="$RD_CONTAINER_ENGINE"
         --kubernetes.enabled=false
     )
+    if [ -n "$RD_CONTAINER_ENGINE" ]; then
```

Comment:
why did you move $RD_CONTAINER_ENGINE away from the list of args? it will always be set, even if not specified, the default value applies, and it's required to avoid the first-run dialog